accelerator/core/proxy_apis.c

```diff
@@ -354,12 +356,12 @@ status_t proxy_api_wrapper(const ta_config_t* const iconf, const iota_client_ser
         break;
 
       default:
-        ta_log_error("%s\n", "SC_HTTP_URL_NOT_MATCH");
-        return SC_HTTP_URL_NOT_MATCH;
+        ta_log_error("%s\n", ta_error_to_string(SC_HTTP_COMMAND_NOT_MATCH));
+        return SC_HTTP_COMMAND_NOT_MATCH;
     }
 
     if (ret) {
-      ta_log_error("error code: %d\n", ret);
+      ta_log_error("%s\n", ta_error_to_string(ret));
```

Comment:
use `ta_log_error("error code = %d: %s\n", ret, error_2_string(ret));` here, since the `ret` is in variable type `retcode_t`, but not `status_t`.

Reply:
This `ret` is assigned from the switch statement above. The return value of `api_xxx` functions seem to return `status_t` (They all return `str_from_char_buffer`)

Reply:
I'll assign `ret` to `SC_HTTP_COMMAND_NOT_MATCH` in the default case.

Reply:
My fault. The current implementation is right